tests/test_functors.py

```diff
@@ -533,6 +534,71 @@ def _testLocalPhotometryFunctors(self, functor, parq, testValues):
                                     atol=1e-13,
                                     rtol=0))
 
+    def testDipPhotometry(self):
+        """Test calibrated flux calculations for dipoles."""
+        fluxNeg = -100
+        fluxPos = 101
+        fluxErr = 10
+        calib = 10
+        calibErr = 1
+
+        absMean = 0.5 * (fluxPos - fluxNeg) * calib
```

Comment:
See style guide about spaces around operators: https://developer.lsst.io/python/style.html#binary-operators-should-be-surrounded-by-a-single-space-except-for